### PR TITLE
Q020 and tsQualityEngine

### LINKED ISSUE #179: Validation (2017): Q020 (Institution Mailing Address)
**Edit #:** Q020
**Type:** Quality
**Category:** Transmittal Sheet (only)
**Column:** Institution Mailing Address

Institution address should not = parent address.

Additional resources:
* [FFIEC's 2016 HMDA File Spec](https://www.ffiec.gov/hmda/pdf/spec2016.pdf)
* [FFIEC's 2016 HMDA Edits](https://www.ffiec.gov/hmda/pdf/edit2016.pdf)
* [CFPB's 2017 HMDA File Spec](http://www.consumerfinance.gov/hmda/static/for-filers/2017/2017-HMDA-File-Specifications.pdf)

### PR DESCRIPTION
**Pull request by kgudel**
closes #179

### CODE REVIEW

File: validation/src/main/scala/hmda/validation/engine/ts/quality/TsQualityEngine.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package hmda.validation.engine.ts.quality
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.api.ValidationApi
+import hmda.validation.engine.ts.TsCommonEngine
+import hmda.validation.rules.ts.quality._
+
+trait TsQualityEngine extends TsCommonEngine with ValidationApi {
```
Comment (nickgrippin) [inline validation/src/main/scala/hmda/validation/engine/ts/quality/TsQualityEngine.scala]: You might need to throw this into the `TsEngine` trait.  I would double-check with @jmarin though.

File: validation/src/test/scala/hmda/validation/engine/ts/quality/TsQualityEngineSpec.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class TsQualityEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with TsQualityEngine {
+
+  override implicit val ec: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
+
+  property("A LAR must pass quality checks") {
```
Comment (schbetsy) [inline validation/src/test/scala/hmda/validation/engine/ts/quality/TsQualityEngineSpec.scala]: `LAR` -> `Transmittal Sheet`

**Comment by schbetsy** [comment]
Quick correction to a property name, then LGTM.

**Comment by nickgrippin** [comment]
LGTM as well.  Feel free to merge as soon as the property name is fixed.

**Pull request merged**